sdcm/utils/distro.py

```diff
@@ -22,30 +22,97 @@ class DistroError(Exception):
     pass
 
 
-# pylint: disable=too-many-public-methods
 @enum.unique
-class Distro(enum.Enum):
-    UNKNOWN = (None, None)
-    CENTOS7 = ("centos", "7")
-    CENTOS8 = ("centos", "8")
-    RHEL7 = ("rhel", "7")
-    RHEL8 = ("rhel", "8")
-    OEL7 = ("ol", "7")
-    OEL8 = ("ol", "8")
-    AMAZON2 = ("amzn", "2")
-    ROCKY8 = ("rocky", "8")
-    DEBIAN8 = ("debian", "8")
-    DEBIAN9 = ("debian", "9")
-    DEBIAN10 = ("debian", "10")
-    DEBIAN11 = ("debian", "11")
-    UBUNTU14 = ("ubuntu", "14.04")
-    UBUNTU16 = ("ubuntu", "16.04")
-    UBUNTU18 = ("ubuntu", "18.04")
-    UBUNTU20 = ("ubuntu", "20.04")
-    UBUNTU21 = ("ubuntu", "21.04")
-    UBUNTU21_10 = ("ubuntu", "21.10")
-    UBUNTU22 = ("ubuntu", "22.04")
-    SLES15 = ("sles", "15")
+class DistroBase(enum.Enum):
+    RHEL = enum.auto()
+    DEBIAN = enum.auto()
+    UNKNOWN = enum.auto()
+
+
+# A list of tupples of the form:
+# (enum_prefix, os name, versions list (format is either "major.minor" or "major"), boolean - debian_like == True else rehl_like)
```

Comment:
Typo in the `rehl_like` part. Better to fix it for case someone searches by `rhel` word.

Reply:
Will change